.github/workflows/binary-package.yml

```diff
@@ -0,0 +1,45 @@
+name: Build Binary Package
+
+on:
+  workflow_dispatch: # run workflow manually from the Actions tab
+  push:
+    branches:
+      - main
+      - dev
+
+jobs:
+  build-package:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Install Node.js 18.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+
+      - name: Install Dependencies
+        run: npm install
```

Comment:
why `npm install` here?

Reply:
Because pnpm-lock.json is not supported by the action component.

See https://github.com/lando/pkg-action/issues/4

Reply:
Honestly speaking, binary package is not what we want. But it doesn't need any additional work, after all.

Reply:
[actions/upload-artifact](https://github.com/actions/upload-artifact) is used to upload something produced by the action. We don't have things to upload.

Reply:
like dist/*?

> [actions/upload-artifact](https://github.com/actions/upload-artifact) is used to upload something produced by the action. We don't have things to upload.

Reply:
> Honestly speaking, binary package is not what we want. But it doesn't need any additional work, after all.

It's important to realize what we need. Building a binary executable is useless. Instead, uploading the compiled Javascript files makes sense, since they can be copied to the server and reproduce the environment. However, the best choice is to wait for our docker file and upload a docker image here.

Reply:
It's harmless after all.

Reply:
Binary executables might be easier for front-end developers to use. They just need to click, then it runs. This is especially useful if another version of node.js is installed on the front-end developers' computer.